handlers/cna_handler/__init__.py

```diff
@@ -0,0 +1,24 @@
+"""Handler for CNA."""
+
+import requests
+from bs4 import BeautifulSoup
+from comment import Comment
+from handlers.abstract_base_handler import AbstractBaseHandler, HandlerError
+
+
+class CNAHandler(AbstractBaseHandler):
+    """Handler for CNA."""
+
+    @classmethod
+    def handle(cls, url):
+        headers = {"user-agent":
```

Comment:
Is there a downside to not specifying user-agent? If so, should user-agent be refactored out so it can be imported and used by all Handlers?

Reply:
During the time that I was writing and testing this handler, CNA checks the user-agent so the scraper needs spoofing to retrieve the article properly. After just testing the parser again, user-agent spoofing is no longer needed now.

> If so, should user-agent be refactored out so it can be imported and used by all Handlers?

It's not necessary to implement user-agent spoofing for all Handlers at the moment but thinking forward, I believe it's good to have it implemented. Who knows what CNA is planning for, when they checked the user-agent during my initial tests.

I can remove the user-agent spoofing from the CNA Handler before you accept this pull request.

Reply:
Okay, sounds good. If it's required, I'd suggest refactoring it out as a variable then, instead of pasting it as a string directly into the headers object.